projects/packages/blocks/composer.json

```diff
@@ -3,11 +3,13 @@
 	"description": "Register and manage blocks within a plugin. Used to manage block registration, enqueues, and more.",
 	"type": "jetpack-library",
 	"license": "GPL-2.0-or-later",
-	"require": {},
+	"require": {
+		"automattic/jetpack-constants": "@dev"
+	},
 	"require-dev": {
 		"automattic/wordbless": "dev-master",
 		"brain/monkey": "2.6.1",
-		"yoast/phpunit-polyfills": "1.1.0",
+		"yoast/phpunit-polyfills": "1.0.4",
```

Comment:
Composer required a valid version.

Reply:
This is odd. [1.1.0 is a valid version](https://github.com/Yoast/PHPUnit-Polyfills/releases/tag/1.1.0), and it's what we use all over the monorepo today.